### DISCUSSION TITLE
People can't be making changes to what they did not create. <title>

### DISCUSSION THREAD

**Discussion by satoshi-MAM** [community/community#43606] [Projects and Issues]
### Select Topic Area

General

### Body

Build together answer me this what have you create?

**Comment by jose19961975** [comment]
Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________
From: Satoshi nakamoto ***@***.***>
Sent: Saturday, January 7, 2023, 9:27 PM
To: community/community ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GitHub Community] People can't be making changes to what they did not create. <title> (Discussion #43606)


Select Topic Area

General    exposed smwloh

Body

Build together answer me this what have you create?

**Comment by jfullstackdev** [comment]
it's not possible without write permission, or else, that's a great security risk
